### PR TITLE
feat(macro): add l10n-zh translation for `{{AvailableInWorkers}}`

### PR DESCRIPTION
**Pull request by yin1999**
## Summary

translation of `Note:` is from <https://github.com/mdn/yari/tree/main/markdown/localizations>

add zh-TW translation of `This feature is available in <a href="/${locale}/docs/Web/API/Web_Workers_API">Web Workers</a>`

### CODE REVIEW

**Comment by yin1999** [comment]
poke @irvin for a review

File: kumascript/macros/AvailableInWorkers.ejs [hunk 1]
```diff
@@ -21,11 +21,14 @@ const note = mdn.localString({
...
     "ko": "주의:",
     "ru": "Примечание:",
+    "zh-CN": "备注：",
+    "zh-TW": "備註："
 });
 
 const textDefault = mdn.localString({
   "en-US": `This feature is available in <a href="/${locale}/docs/Web/API/Web_Workers_API">Web Workers</a>`,
   "zh-CN": `此特性在 <a href="/${locale}/docs/Web/API/Web_Workers_API">Web Worker</a> 中可用`,
+  "zh-TW": `此特性在<a href="/${locale}/docs/Web/API/Web_Workers_API">Web Worker</a>中可用`,
```
Comment (irvin) [inline kumascript/macros/AvailableInWorkers.ejs:31]:
```suggestion
  "zh-TW": `此功能可在 <a href="/${locale}/docs/Web/API/Web_Workers_API">Web Worker</a> 中使用`,
```

**Comment by schalkneethling** [comment]
@irvin Looks like this one is ready to merge?

**Comment by irvin** [comment]
> @irvin Looks like this one is ready to merge?

yes please

**Review by schalkneethling — APPROVED** [review]
Thank you, @yin1999 🎉

**Pull request merged**